### ISSUE TITLE
create gem fails if nextflow.config parameter has space in it

### ISSUE THREAD

**Issue by JohnHadish** [SystemsGenetics/GEMmaker#169]
If the `nextflow.config` parameter `machine_name` has a space character in it, the create gem process will fail. This is because `create-gem.py` uses this parameter to write the output name of the file. If there are spaces in the file, nextflow will not be able to find it, and will throw an error.

It is not evident that this parameter is used in this way if the user is just looking at the `nextflow.config`, as this portion appears to be meta data. For example, the following example will fail:
```
params {
  project {
    name = "Gala Post Harvest Treatment Experiment"
    machine_name = "Kamiak Ficklin Cores"
    description = "See README.md"
  }
```

while this will run:
```
params {
  project {
    name = "Gala Post Harvest Treatment Experiment"
   machine_name = "Kamiak"
    description = "See README.md"
  }
```

I am of the opinion that this parameter should not be used to create file names. The other option would to require that the `machine_name` parameter have underscores.

**Comment by spficklin** [comment]
This should be resolved by PR #205 .  The machine name is no longer used.

**Comment by spficklin** [comment]
Fixed with PR #205.  The `machine_name` parameters is no longer present.
